code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -79,6 +104,7 @@
 	handle_stagger() // 1 each time
 	handle_slowdown() // 0.4 each time
 	handle_halloss() // 3 each time
+	handle_stealth()
```

Comment:
`/mob/living/carbon/Xenomorph/Hunter/handle_status_effects()`

Reply:
This has been done